trepb/main.go

```diff
@@ -4,21 +4,15 @@ import (
 	"encoding/json"
 	"flag"
 	"fmt"
-	"log"
 	"os"
 	"time"
 
 	"github.com/dadosjusbr/storage"
-	"github.com/joho/godotenv"
```

Comment:
Uma vez que você alterou essas dependência, deve precisar alterar o go.mod e o go.sum locais. Isso é importante para quem não for usar com docker.